### PR TITLE
Journal Bundle and Journal API

### PR DESCRIPTION
**Pull request by bessiesteinberg**
Create Journal Bundle GET api

### CODE REVIEW

File: course_discovery/apps/journal/api/serializers.py [hunk 1]
```diff
@@ -21,3 +25,25 @@ class Meta(object):
...
+
+    class Meta:
+        model = JournalBundle
+        fields = (
+            'uuid',
+            'title',
+            'partner',
+            'journals',
+            'courses'
+        )
```
Comment (bill-filler) [inline course_discovery/apps/journal/api/serializers.py]: NIT: add newline at end of file

File: course_discovery/apps/journal/constants.py [hunk 2]
```diff
@@ -0,0 +1 @@
+UUID_PATTERN = '[0-9a-f-]+'
```
Comment (bill-filler) [inline course_discovery/apps/journal/constants.py]: NIT: add newline at end of file

File: course_discovery/apps/journal/models.py [hunk 3]
```diff
@@ -54,5 +54,9 @@ class JournalBundle(TimeStampedModel):
         max_length=CHARFIELD_MAX_LENGTH,
         unique=True
     )
+    partner = models.ForeignKey(Partner)
```
Comment (bill-filler) [inline course_discovery/apps/journal/models.py:57]: this seems fine. does programs bundle also have a link to Partner?

**Comment by bill-filler** [comment]
@BessieSteinberg 
nice work! few nits then good to merge

**Review by bill-filler — APPROVED** [review]
approved

File: course_discovery/apps/journal/models.py (hunk 3 above)
Comment (bessiesteinberg) [reply course_discovery/apps/journal/models.py:57]: yeah Programs does

**Pull request merged**